client/app/bundles/course/assessment/question/programming/components/ProgrammingQuestionForm.jsx

```diff
@@ -19,6 +19,8 @@ import ChipInput from '../../../../../../lib/components/ChipInput';
 
 import styles from './ProgrammingQuestionForm.scss';
 import translations from './ProgrammingQuestionForm.intl';
+import { DEFAULT_TIME_LIMIT } from '../../../constants';
```

Comment:
I think we should put the constants at top of the component to make it self-contained. The `constants.js` are currently used for redux actions which are shared between multiple places.